### PR TITLE
Creating Blockquote component

### PR DESCRIPTION
**Pull request by henriquecfreitas**
Resuming discussions from #93 

Pending fixes:

- [x] Alignment of the quote icon (right aligned text)
- [x] Should the quote icon animate?
- [x] Adjust RichText to not include headings
- [x] Apply appropriate default font-sizing

### CODE REVIEW

**Comment by quaardvark** [comment]
@henriquecfreitas Let's not forget to adjust RichText to not include headings and to apply appropriate default font-sizing (overridable by labels)

**Comment by henriquecfreitas** [comment]
> @henriquecfreitas Let's not forget to adjust RichText to not include headings and to apply appropriate default font-sizing (overridable by labels)

Thank you, just added on the pending fixes list (feel free to edit it if you feel like we're missing anything else)
I'll attack it as soon as I finish the work on Article List component

**Comment by henriquecfreitas** [comment]
@quaardvark @pkrawc this is ready for review now

File: src/types/info.json [hunk 1]
```diff
@@ -165,7 +165,7 @@
               "content" : {
                 "type" : "StructuredText",
                 "config" : {
-                  "single" : "heading1, heading2, heading3, heading4, heading5, heading6, strong, em",
+                  "single": "paragraph, strong, em",
```
Comment (quaardvark) [inline src/types/info.json:168]: I think it should allow multiple nodes

Comment (pkrawc) [reply src/types/info.json:168]: Agreed

File: src/components/organisms/Blockquote/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <Box as={motion.aside} {...animation?.container} {...props}>
+      <Container ref={intersectionView} maxWidth={containerWidth} py="6rem">
+        <Box sx={{ position: "relative" }}>
+          <Box
+            as={motion.div}
+            {...animation?.item}
+            sx={{
+              position: "absolute",
+              width: "7rem",
+              [alignment]: "2rem",
```
Comment (pkrawc) [inline src/components/organisms/Blockquote/index.js:30]: A clever way to do this to accommodate for both text aligments.

**Review by pkrawc — APPROVED** [review]
With the change that Dima requested, this looks good to me.

**Review by pkrawc — APPROVED** [review]

**Pull request merged**